metadata-ingestion/tests/integration/mode/test_mode.py

```diff
@@ -142,8 +170,89 @@ def test_mode_ingest_failure(pytestconfig, tmp_path):
             }
         )
         pipeline.run()
-        try:
```

Comment:
Previously this wasn't actually testing anything... `raise_from_status` wasn't raising exceptions (we were overriding the implementation on the response object...) so we wouldn't assert anything. Changed up this file a bit so this test is actually testing stuff